lib/presentation/home_page/home_page.dart

```diff
@@ -29,7 +32,17 @@ class _HomePageState extends State<HomePage> {
               Navigator.of(context)
                   .pushNamed(SettingsPage.settingsPageRouteName);
             },
-            icon: const Icon(Icons.settings),
+            icon: const DecoratedIcon(
+              decoration: IconDecoration(
+                border: IconBorder(
+                  color: Colors.black26,
+                  width: 5,
```

Comment:
Try to use same border's width everywhere. Here you've got width 5, earlier it was default value, it looks better when details are everywhere the same.

Reply:
Ok I changed the bordered width, but in one icon i still need to stay with default value, because, It's  arrow icon and it's more tinier than another icon (this arrow is responsible for displaying wind degree,) so when i set width to 5 that arrow icon border isn't displaying properly :D

Reply:
Ok got it : D